Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM centos:centos6
 
-RUN yum update -y
-RUN yum install wget -y
-RUN yum install tar -y
-RUN yum groupinstall "Development tools" -y
+RUN \
+    yum update -y &&\
```

Comment:
it's picky, but the Docker best practices document suggests putting a space before the backslash for readability.